api/kroxylicious-filter-api/src/main/templates/Kproxy/SpecificFilterArrayInvoker.ftl

```diff
@@ -38,6 +38,7 @@ import org.apache.kafka.common.protocol.ApiMessage;
  * Invoker for KrpcFilters that implement any number of Specific Message interfaces (for
  * example {@link io.kroxylicious.proxy.filter.AlterConfigsResponseFilter}.
  */
+@SuppressWarnings("SwitchStatementWithTooFewBranches")
```

Comment:
I don't understand why this is needed (but I'm only reading the diff of a template used to generate code, so I'm easily confused). Could we make it more localised?

Reply:
IntelliJ is confused by the the template. As it only sees the default case in the switch so highlights it as having too few branches however the generated code has the request number of cases. The inspection would never trigger on the generated code as we include the default case so even if we somehow lost an entry in the germination process it wouldn't pick it up.

I've moved it from class level to each of the 4 methods which have the switch (I initially had it at class level to avoid the duplication)